### ISSUE TITLE
Does this work support navigation while building maps?

### ISSUE THREAD

**Issue by D-jojo** [concept-graphs/concept-graphs#77]
Thank you for publishing such a great work.
I would like to ask if this work supports navigation while building a map, because the application scenario I want to do is that the robot builds a map in the environment and finds the target object according to human instructions.
Looking forward to your reply！

**Comment by georgegu1997** [comment]
Yes, it should work with some modifications. ConceptGraphs framework itself is mostly a mapping system, and it can be integrated into a navigation pipeline quite easily. We have demonstrated this in an offline manner

The major consideration to get it online should be the latency. Our original implementation was not real-time, mainly because of the SAM dense segmentation process. The revised implementation in the `ali-dev` branch is now in real time, and it should be useable for navigation while mapping. 

Closing this issue for now. Feel free to re-open if need.
